### PR TITLE
Fixed being able to land the white ship on the transit z level

### LINKED ISSUE #41329: Somehow Land White Ship on Transit Z-Level (Metastation)
Issue reported from  Round ID: 96667 (/tg/Station Bagil [ENGLISH] [US-EAST] [100% FREE LAG])

Reporting client version: 512

As stated above, I somehow was able to land the White Ship in the transit Z-Level, I landed just above the Arrivals Shuttle but when I exited the ship I was flung off akin to leaving a ship mid-transit.

Steps to reproduce/Exactly what I did

1: Land the ship on Lavaland
2: Take off via sending the ship to a "deep space" location
3: While mid-transit, enter the 'custom ship landing console' thing. From there I was able to set a location in the transit z-level
4: Set your ship to land somewhere in the Transit Z-Level
5: Profit

### PR DESCRIPTION
**Pull request by Cruix**
Fixes #41329

### CODE REVIEW

File: code/modules/shuttle/white_ship.dm [hunk 1]
```diff
@@ -44,6 +44,12 @@
 /obj/machinery/computer/camera_advanced/shuttle_docker/whiteship/Initialize()
```
Comment (Dennok) [inline code/modules/shuttle/white_ship.dm]: it need in 
/obj/machinery/computer/camera_advanced/shuttle_docker/Initialize()

**Comment by Dennok** [comment]
Dont fully fixed.

**Comment by Dennok** [comment]
No. All shuttles use transit to travel. And caravan ship have shuttle_docker/caravan console.

**Comment by Dennok** [comment]
abbordage in transit
abbordage in transit s

**Comment by Cruix** [comment]
Then the caravan ambush ruin should be locked to its own z-level, or if it uses the same behavior as the white ship, it should inherit from the white ship computer.

**Comment by Dennok** [comment]
i like your resolve
i just move it to more basic console level #41475

**Comment by Cruix** [comment]
Apparently someone made yet another docking computer that is explicitly not locked to it's z-level. No reason for me to fix it now, as someone else already has a PR up for it.

**Comment by Dennok** [comment]
there is no another docking computer, no one docking computer locked to one z level by hardcode. It has a list of possible places what can be accesed.

**Pull request merged**